virtualsmartcard/src/vpicc/virtualsmartcard/CryptoUtils.py

```diff
@@ -19,6 +19,8 @@
 import sys, binascii, random, logging
 from struct import pack
 from binascii import b2a_hex, a2b_hex
+from base64 import b64encode
```

Comment:
The `b64encode()` function was part of the `PBKDF2` implementation we included from a third party as well, so I replaced it by using the equivalent stdlib function. It's not used anywhere else in the code.
Right now the only thing left from the borrowed `PBKDF2` implementation are the `crypt()` and `_makesalt()` functions. Maybe I will do some more changes to these functions as well but I will have to think a bit more about it in order to avoid braking stuff.